### PR TITLE
Add regions to investing

### PR DESCRIPTION
**Pull request by ApCoder123**

### CODE REVIEW

File: entries/p/prudential.com.json [hunk 1]
```diff
@@ -1,12 +1,14 @@
 {
   "Prudential Financial": {
     "domain": "prudential.com",
-    "url": "https://www.prudential.com",
```
Comment (Carlgo11) [inline entries/p/prudential.com.json]: This line needs to be here.

Comment (phallobst) [reply entries/p/prudential.com.json]: I'm afraid that's the same problem as in #6200 - neither URL does work.

Comment (Carlgo11) [reply entries/p/prudential.com.json]: https://prudential.com didn't work for me anyway :shrug:

I'm not sure how to debug the failed test. For starters I can't find what DNS servers the GitHub workflow uses :thinking:

File: entries/i/ig.com.json [hunk 2]
```diff
@@ -1,14 +1,16 @@
 {
```
Comment (Carlgo11) [inline entries/i/ig.com.json:1]: Is it worth listing these countries?

Comment (ApCoder123) [reply entries/i/ig.com.json:1]: I didn't include those, as it did seem to allow registrations from any country - see "IG International" at the bottom of that page and https://www.ig.com/en/application-form.

File: entries/p/prudential.com.json (hunk 1 above)
Comment (Carlgo11) [reply entries/p/prudential.com.json]: After much debugging and some reverse shells into the github workflow I've found that the problem is indeed due to Microsoft Azure's DNS servers. Setting the nameservers to `1.1.1.1` in `/etc/resolv.conf` fixes the problem. Oddly enough it only happens when running the command inside docker on Microsoft Azure :thinking:

**Review by Carlgo11 — APPROVED** [review]

File: entries/w/wealthsimple.com.json [hunk 3]
```diff
@@ -10,6 +10,10 @@
     "documentation": "https://help.wealthsimple.com/hc/en-ca/articles/360058451253",
     "keywords": [
       "investing"
+    ],
+    "regions": [
+      "gb",
```
Comment (phallobst) [inline entries/w/wealthsimple.com.json]: See https://help.wealthsimple.com/hc/en-gb/articles/4411041188891
```suggestion
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**